tools/topology/topology2/include/pipelines/cavs/passthrough-playback.conf

```diff
@@ -53,8 +53,10 @@ Class.Pipeline."passthrough-playback" {
 			node_type $HDA_HOST_OUTPUT_CLASS
 			type	"aif_in"
 			num_audio_formats 2
-			# 16-bit 48KHz 2ch
+			# 16-bit input format 48KHz 2ch. Output sample format is always 32-bit for playback
 			Object.Base.audio_format.1 {
+				out_bit_depth		32
+				out_valid_bit_depth	32
```

Comment:
The patch looks good, I just wonder if the assumption of 48kHz is always correct. I am not sure how e.g. we might handle BT offload which operates at 8 or 16 kHz and it's mono if I am not mistaken?

Or may we punt and deal with this later when we have SSP support?

Reply:
right, we'll have to deal with it when we get there.